instrumentation/jboss-logmanager/jboss-logmanager-mdc-1.1/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jbosslogmanager/mdc/v1_1/JbossLogmanagerMdcTest.java

```diff
@@ -90,11 +93,28 @@ void idsGeneratedWhenSpanProvided() throws NoSuchMethodException {
 
     assertThat(logRecords.size()).isEqualTo(3);
 
+    Method getMdcCopy = null;
+    boolean hasGetMdcCopy = false;
+    try {
+      getMdcCopy = logRecords.get(0).getClass().getMethod("getMdcCopy");
+      hasGetMdcCopy = getMdcCopy != null;
```

Comment:
`getMdcCopy != null` is always true at this point - consider removing the boolean flag and just checking for `getMdcCopy != null` in the conditions below.

Reply:
updated, thanks!